src/openforms/submissions/views.py

```diff
@@ -192,6 +194,46 @@ def get_form_resume_url(self, submission: Submission) -> str:
         )
 
 
+class DownloadSubmissionReportView(PrivateMediaView):
+    """
+    Download the PDF report containing the submission data.
+
+    This URL requires a token which is tied to the submission from the session. The
+    token automatically expired after
+    ``settings.SUBMISSION_REPORT_URL_TOKEN_TIMEOUT_DAYS`` days.
+    """
+
+    queryset = SubmissionReport.objects.all()
+    pk_url_kwarg = "report_id"
+    file_field = "content"
+    raise_exception = True
+
+    object: SubmissionReport
+
+    def has_permission(self):
+        report: SubmissionReport = self.get_object()
+        token = self.kwargs["token"]
+        return submission_report_token_generator.check_token(report, token)
+
+    def get_sendfile_opts(self) -> dict:
+        path = Path(self.object.content.name)
+        opts = {
+            "attachment": True,
+            "attachment_filename": path.name,
+            "mimetype": "application/pdf",
+        }
+        return opts
+
+    def get(self, request: HttpRequest, *args, **kwargs):
+        self.object = self.get_object()
+
+        response = super().get(request, *args, **kwargs)
+
+        self.object.last_accessed = timezone.now()
```

Comment:
Do we need local time here, like `datetime_in_amsterdam` ?And basically is there a standard for when we need the local time?

Reply:
I think the difference between using `timezone.now` vs `datetime_in_amsterdam` is whether you want the datetime displayed (i.e. string formatted or not). `last_accessed` is a db field and they are stored with UTC timezones so it's good to use `timezone.now()`

Sergei might confirm

Reply:
no, django converts everything to UTC anyway when it's being saved in the DB. The `datetime_in_amsterdam` is only needed in edge cases:

* presentation to the end-use, usually this is handled already in the django template layer automatically
* when interfacing with other systems that don't use ISO-8601 datetimes and expect naive, local times

So usually these are external factors outside of our control.